extensions/wikia/AdEngine/js/lookup/lookupFactory.js

```diff
@@ -0,0 +1,90 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.lookupFactory', [
+	'ext.wikia.adEngine.adTracker',
+	'wikia.log'
+], function (adTracker, log) {
+	'use strict';
+
+	function create(module) {
+		var called = false,
+			response = false,
+			timing;
+
+		function call(skin) {
+			log('call', 'debug', module.logGroup);
+
+			if (!Object.keys || (typeof module.isEnabled === 'function' && !module.isEnabled())) {
+				log(['call', 'Module is not enabled/supported', module.name], 'debug', module.logGroup);
```

Comment:
This is ambiguous and misleading. Let's have two conditions instead. One will log that IE8 is not supported and the other one will log that a module is not enabled.